user-journey.adoc

```diff
@@ -1,62 +1,91 @@
-= User Journey for Istio/Jaeger
+= User Journey - Installing Istio + Fabric8 Launcher
 
-This document describes the steps required to install a local Istio Openshift instance, necessary prerequisites, and describes how the your can experiment with Istio enabled missions using the launcher.
+This document describes how to use the Istio installer to install the following components:
 
-== Flows description
+* Istio platform to connect, manage, and secure microservices.  This Istio installation includes the following components:
+** Elasticsearch
+** Grafana
+** Istio
+** Jaeger
+** Prometheus
+* Fabric8 integrated development platform launcher.
+* Four Istio-specific sample applications (boosters) and tutorials (missions) that let you explore some of the Istio features.
 
-The Fabric8 Launcher provides two workflows to play with an Istio mission (zip vs. git flow), and this document will apply to either - described below:
+== Platform launcher workflows
```

Comment:
Platform Launcher? Is that what it's called?

Reply:
That's one of my questions, I'm not familiar with all the developer platform/RHOAR terminology.